test/integration/features/cmd-hostfolder.feature

```diff
@@ -0,0 +1,50 @@
+@cmd-hostfolder @command
+
+Feature: hostfolder commands
+  As a user I can perform basic operations of Minishift with hostfolder mount
+
+  Scenario: User should not be able to list hostfolder when it does not exist
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift hostfolder list" succeeds
+     Then stdout should contain
+     """
+     no host folders defined
+     """
+  Scenario: User should be able to add a hostfolder of type SSHFS
+    When executing "minishift start" succeeds
+     Then Minishift should have state "Running"
+    When executing "minishift hostfolder add -t sshfs --source /home/minishift_ci --target /mnt/sda1/myshare myshare" succeeds
+    And executing "minishift hostfolder list" succeeds
+     Then stdout should contain
+     """
+     "myshare\s* sshfs\s* /home/minishift_ci\s* /mnt/sda1/myshare\s* N"
+     """
+    When executing "minishift hostfolder mount myshare" succeeds
+    And executing "minishift hostfolder list" succeeds
+     Then stdout should contain
+     """
+     "myshare\s* sshfs\s* /home/minishift_ci\s* /mnt/sda1/myshare\s* Y"
+     """
+    When executing "echo 'hello' > /home/minishift_ci/test_sshfs" succeeds
```

Comment:
This will fail as the "executing" step does not execute directly in shell, but instead uses `minishift` or `oc` binaries directly. We will need to add a specialized step which will be able to write to files I think.